extensions/ql-vscode/src/view/common/Alert.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+type ContainerProps = {
+  type: 'warning' | 'error';
+  inverse?: boolean;
+};
+
+const getBackgroundColor = ({ type, inverse }: ContainerProps): string => {
+  if (!inverse) {
+    return 'var(--vscode-notifications-background)';
+  }
+
+  switch (type) {
+    case 'warning':
+      return 'var(--vscode-editorWarning-foreground)';
+    case 'error':
+      return 'var(--vscode-debugExceptionWidget-border)';
+  }
+};
+
+const getTextColor = ({ type, inverse }: ContainerProps): string => {
+  if (!inverse) {
+    return 'var(--vscode-tab-activeForeground)';
+  }
+
+  switch (type) {
+    case 'warning':
+      return 'var(--vscode-panel-background)';
```

Comment:
```suggestion
      return 'var(--vscode-editor-background)';
```
Doesn't actually change the colour but would fit in since I'm suggesting other `editor` colours elsewhere 🤷